numba/tests/test_typedlist.py

```diff
@@ -106,6 +106,17 @@ def test_basic(self):
         # index
         self.assertEqual(l.index(15), 4)
 
+    def test_list_extend_unicode_type(self):
+        @njit
+        def foo(string):
+            l = List()
+            l.extend(string)
+            return l
+
+        for func in (foo, foo.py_func):
+            for string in ("abc", "\nabc\t"):
```

Comment:
This probably needs an empty string and a singleton (one character string). I saw you other commit about fixing `extend` for empty iterables, so probably just adding a single `a` will be fine.

Reply:
Done :)